.github/workflows/release-build-outputs.yml

```diff
@@ -15,7 +15,7 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   CHANGELOG_FILE_NAME: ./docs/CHANGELOG.md
-  RELEASE_INCLUDED: $CHANGELOG_FILE_NAME README.md LICENSE src .classpath .project pom.xml build.xml ./scripts/jvmrun
+  RELEASE_INCLUDED: $CHANGELOG_FILE_NAME README.md LICENSE src .classpath .project pom.xml build.xml ./scripts/jvmrun ./config/ConnectorConfig.json
```

Comment:
It may be worth looking in to updating this connector to use a `starting-files` folder eventually. For example, the starter project follows that structure: https://github.com/hms-networks/sc-java-maven-starter-project, as well as https://github.com/hms-networks/flexy-cumulocity-connector

Reply:
a good suggestion for next time. I'll make a new issue for this